instructions/index.html

```diff
@@ -31,10 +31,20 @@ <h3>Experience</h3>
   </ul></p>
 </section>
 
+<section id="skills">
+  <h2>Skills</h2>
+  <ul></ul>
+</section>
+
+
 <section>
   <h3>Connect</h3>
   <a href="https://github.com/roselynn411">Github Account</a>
   <p>My email is roselynnfrempong@gmail.com. My Instagram is rose.lynnn</p>
+  <footer id="copyright"> 
```

Comment:
Minor thing but since the inner paragraph element contains the copyright I'd give the footer itself a different id just to keep the intent clear. `<footer id="footer">` would work fine.